controller/api/destination/endpoint_translator_test.go

```diff
@@ -75,6 +83,14 @@ var (
 								Name:  envInboundListenAddr,
 								Value: "[::1]:4143",
 							},
+							{
+								Name:  envAdminListenAddr,
+								Value: "[::1]:4191",
+							},
+							{
+								Name:  envControlListenAddr,
+								Value: "[::1]:4190",
```

Comment:
Probably not affecting the test, but we should have here IPv6 wildcard addresses, which are `[::]`